api/v1alpha1/hostedcluster_types.go

```diff
@@ -406,26 +407,27 @@ type DNSSpec struct {
 
 // ClusterNetworking specifies network configuration for a cluster.
 type ClusterNetworking struct {
-	// ServiceCIDR is...
+	// MachineNetwork is the list of IP address pools for machines.
+	// Default is 10.0.0.0/16 for all platforms other than libvirt.
```

Comment:
The default comes from the installer - if we don't provide it, the installer defaults that in.

The comment is copied directly from the installer type definition. It may no longer be accurate.

Reply:
Okay. removed the incorrect comments. We need to clean up how this is defaulted someday, since it seems to be platform-specific. But it's not critical.